README.md

```diff
@@ -1,48 +1,28 @@
 
-The Iterative Provider makes it easy to:
+- **Orchestrate Resources**: create cloud compute & storage resources without reading pages of documentation
+- **Sync & Execute**: move data & run code in the cloud with minimal configuration
+- **Low cost**: auto-recovery from spot/preemptible instances to vastly reduce cost
+- **No waste**: auto-cleanup unused resources
+- **No lock-in**: switch between cloud vendors with ease
 
-- Rapidly move local machine learning experiments to a cloud infrastructure
-- Take advantage of training models on spot instances without losing any progress
-- Unify configuration of various cloud compute providers
-- Automatically destroy unused cloud resources (compute instances are terminated on job completion/failure, and storage is removed when results are downloaded)
+Iterative's Provider is a [Terraform](https://terraform.io) plugin built with machine learning pipelines in mind. It enables full lifecycle management of computing resources (including GPUs) from several cloud vendors:
 
-The Iterative Provider can provision resources with the following cloud providers and orchestrators:
-
-- Amazon Web Services
+- Amazon Web Services (AWS)
 - Microsoft Azure
-- Google Cloud Platform
-- Kubernetes
-
-## Documentation
-
-See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide to learn how to use the Iterative Provider. More details on configuring and using the Iterative Provider are in the [documentation](https://registry.terraform.io/providers/iterative/iterative/latest/docs).
+- Google Cloud Platform (GCP)
+- Kubernetes (K8s)
 
-## Support
+With a minimal configuration unified across cloud vendors, the aim is to easily move local experiments to the cloud, transparently resume from interrupted low-cost spot instances, and avoid being charged for unused cloud resources (terminate compute instances upon job completion/failure, and remove storage upon download of results).
 
-Have a feature request or found a bug? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.gg/bzA6uY7); we'll be happy to help you get started!
-
-## License
+## Usage
 
-Iterative Provider is released under the [Apache 2.0 License](https://github.com/iterative/terraform-provider-iterative/blob/master/LICENSE).
-
-## Development
-
-### Install Go 1.17+
-
-Refer to the [official documentation](https://golang.org/doc/install) for specific instructions.
-
-### Clone the repository
-
-```console
-git clone https://github.com/iterative/terraform-provider-iterative
-cd terraform-provider-iterative
-```
+See the [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide for a more detailed guide.
 
 ### Install the provider
```

Comment:
Why we should start/focused on contributors not users? Let's focus on users.

Reply:
Exactly - move the contributor section to the bottom. Forgot to update this bit though :facepalm:

Reply:
Minified & corrected.